HousingSearchListener/V1/UseCase/Interfaces/IIndexTransactionUseCase.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace HousingSearchListener.V1.UseCase.Interfaces
+{
+    public interface IIndexTransactionUseCase : IMessageProcessing
+    {
```

Comment:
This interface should not be empty it should have this method in it:
`Task ProcessMessageAsync(EntityEventSns message)`

Reply:
This method is already there in base interface and if you compare with te other use cases interfaces it also does not have this method defined again.